tc/sgx/trusted_worker_manager/enclave/enclave_utils.cpp

```diff
@@ -41,7 +41,7 @@ void Log(int level, const char* fmt, ...) {
     va_start(ap, fmt);
     vsnprintf_s(buf, BUFSIZ, fmt, ap);
     va_end(ap);
-#ifdef DEBUG
+#ifdef TCF_ENCLAVE_LOG
```

Comment:
See comment above about preventing logs in the release build. It makes sense to change this macro to be 
#ifdef DEBUG
    #ifdef TCF_ENCLAVE_LOG
         ocall_Log(level, buf);
    #endif // TCF_ENCLAVE_LOG
#endif  // DEBUG